doc/examples/encryption.rst

```diff
@@ -787,3 +787,23 @@ An application using Azure credentials would look like, this time using
     coll.insert_one({"encryptedField": "123456789"})
 
 The driver will `acquire an access token <https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token>`_ from the Azure VM.
+
+
+.. _CSFLE rewrap_many_data_key:
+
+
+Re-Wrapping Data Keys with CMK
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :ref:`~pymongo.encryption.ClientEncryption.rewrap_many_data_key` utility method allows you to easily
+re-encrypt all of your data-keys with a new CMK, or master key. Note that this does *not* require re-encrypting
+any of the data in your encrypted collections, but rather refreshes the key that protects the keys that encrypt the data::
```

Comment:
Since this section is so small, can we put it inline in the rewrap_many_data_key docstring?